test/hotspot/jtreg/compiler/vectorization/TestNumberOfContinuousZeros.java

```diff
@@ -23,6 +24,7 @@
 
 /**
 * @test
+* @bug 8297172 8331993 8349637
```

Comment:
Drive-by comment.
The `@requires` below are really a shame. This test now is not run in other places, so we would not catch issues elsewhere.

Reply:
I was thinking about refactoring the test with this in mind and using the `Generators` api for creating random values, but I thought that if we were planning on backporting this to older releases (since this bug affects 21 and 24 too) it might be better to do it separately to keep the diff small. What do you think?